### PR TITLE
Add link to Grafana Agent Configuration Generator.

### PR DESCRIPTION
**Pull request by mattdurham**
Add link to the awesome agent configurator

### CODE REVIEW

**Comment by mattdurham** [comment]
@theSuess please review and feel free to suggest changes.

File: docs/sources/flow/_index.md [hunk 1]
```diff
@@ -58,6 +58,11 @@ prometheus.remote_write "default" {
 }
 ```
 
+## Online configurator
+
+Check out the [Grafana Agent Configuration Generator](https://grafana.github.io/agent-configurator/) to get a head start on creating
+flow code. Note that this is experimental and not all components are supported.
+
```
Comment (clayton-cornell) [inline docs/sources/flow/_index.md:65]:
```suggestion
## Grafana Agent configuration generator

The [Grafana Agent configuration generator](https://grafana.github.io/agent-configurator/) will help you get a head start on creating flow code.
{{% admonition type="note" %}}
This feature is experimental, and it does not support all River components.
{{% /admonition %}}

```

**Review by clayton-cornell — COMMENTED** [review]
Did a little tweaking to the info presented. Move the note to the admonition shortcode format, reworded the heading title, etc.

**Review by clayton-cornell — APPROVED** [review]
Tested in local doc build. all LGTM

@mattdurhamwhat versions should we backport to? Just 0.35?

**Review by theSuess — APPROVED** [review]
:rocket:

**Comment by mattdurham** [comment]
I think v0.35 is sufficient, that would be the default most people would see.

**Pull request merged**